modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -15,6 +15,7 @@ In the table, features are marked with the following statuses:
 |*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
 
 |*Version*|*kam*    |*Helm*  |*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*     |*RH SSO* |
+|1.5.0    |0.0.41 TP|3.8.0 GA|4.4.1 GA   |2.3.3 GA |0.2.0 TP        |2.30.3 GA |7.4.0 GA |4.7-4.10
```

Comment:
Please correct the versions.

ApplicationSets - 0.4.1 not 0.2.0
RH-SSO - 7.5.1 not 7.4.0
Operator - 4.8-4.10 not 4.7-4.10